<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit_MatrizResultado">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>361</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Dados da Matriz A&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Adicao">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ASubtracaoB">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>A - B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AMultiplicacaoB">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string> A * B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TranspostaA">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Transposta de A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PotenciacaoA">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>420</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Potenciação em A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AMultiplicacaoK">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>480</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>A * k</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CriarMatrizA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>CRIAR</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>260</y>
      <width>361</width>
      <height>301</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>É Triangular Superior?</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>É Triangular Inferior?</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>É Simétrica?</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>É Identidade?</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>São Iguais?</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>São Diferentes?</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>É Ortogonal?</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>É Nula?</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>É de Permutação?</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Matriz A</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Matriz B</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>211</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Linhas:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Colunas:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLineEdit" name="lineEdit_LinhasMatrizB"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_ColunasMatrizB"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Dados da Matriz B&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CriarMatrizB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>CRIAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonExecutarGrid">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>520</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>EXECUTAR GRID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_BMultiplicacaoK">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>480</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>B * k</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_BSubtracaoA">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>B - A</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>211</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Linhas:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Colunas:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit_LinhasMatrizA"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_ColunasMatrizA"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit_SaidaMatrizA">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_SaidaMatrizB">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_BMultiplicacaoA">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>B * A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TranspostaB">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>390</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Transposta de B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PotenciacaoB">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>450</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Potenciação em B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ApagarMatrizA">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>140</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>APAGAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ApagarMatrizB">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>APAGAR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>732</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
